irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -783,6 +784,32 @@ namespace iroha {
               ELSE 1
           END AS result)";
 
+    const std::string PostgresCommandExecutor::compareAndSetAccountDetailBase = R"(
+          PREPARE %s (text, text, text, text, text) AS
+          WITH %s
+              inserted AS
+              (
+                  UPDATE account SET data = jsonb_set(
+                  CASE WHEN data ?$1 THEN data ELSE
+                  jsonb_set(data, array[$1], '{}') END,
+                  array[$1, $3], $4::jsonb) WHERE
+                  CASE WHEN data ?$1 AND data->$1 ?$3 THEN
+                  data->$1->$3=$5::jsonb ELSE TRUE END AND account_id=$2 %s
```

Comment:
>ELSE TRUE END AND

please reformat if possible to improve readability

Reply:
Fixed